### PR TITLE
Support installing pkgconfig file with CMake

### PR DESCRIPTION
**Pull request by MehdiChinoune**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -451,6 +451,17 @@ set_target_properties(flint
     RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/bin"
 )
 
+# Install PkgConfig file
+set(prefix ${CMAKE_INSTALL_PREFIX})
+set(exec_prefix "\$\{prefix\}")
+set(includedir "\$\{prefix\}/include/flint")
```
Comment (albinahlback) [inline CMakeLists.txt:457]: That's supposed to be just `${prefix}/include`, no?

Comment (MehdiChinoune) [reply CMakeLists.txt:457]: Yes, the result in pkgconfig file should be like this. That's Why I put backslash before `$` and `{` so It doesn't replaced by the variable above.

Comment (albinahlback) [reply CMakeLists.txt:457]: I mean, it's not supposed to be suffixed by `/flint`.

Comment (MehdiChinoune) [reply CMakeLists.txt:457]: Take a look at flin.pc.im

Comment (albinahlback) [reply CMakeLists.txt:457]: For reference, this is my `flint.pc`:
```
prefix=/home/albin/cbuilds
exec_prefix=${prefix}
includedir=${prefix}/include
libdir=${exec_prefix}/lib

Name: FLINT
Description: Fast Library for Number Theory
Version: 3.1.0-dev
URL: https://flintlib.org/
Requires: gmp >= 6.2.1 mpfr >= 4.1.0
Cflags: -I${includedir}
Libs: -L${libdir} -lflint
```

Comment (MehdiChinoune) [reply CMakeLists.txt:457]: You mean users are supposed to include flint headers like `#include "flint/flint.h"` instead of `#include "flint.h"`

Comment (albinahlback) [reply CMakeLists.txt:457]: Yes. See https://flintlib.org/doc/building.html#linking-and-running-code

Comment (MehdiChinoune) [reply CMakeLists.txt]: fixed.

**Comment by albinahlback** [comment]
Thanks!

**Comment by orlitzky** [comment]
I'm a few days late (sorry), but I think this should be installing to `$libdir/pkgconfig` instead of `lib/pkgconfig`.

**Comment by albinahlback** [comment]
Yeah, sounds right.

**Pull request merged**